assets/scss/base/_buttons.scss

```diff
@@ -77,6 +77,9 @@ button, .button {
   text-decoration: underline;
   color: $link-colour;
   border: none;
+  margin-top: em(10);
+  text-align: center;
+  box-sizing: border-box;
```

Comment:
feels a bit odd that we are not doing this globally as in something like this in `_base.scss`
```
*, *:before, *:after {
  box-sizing: border-box;
}
```

but thats beyond this PR. Maybe raise an issue to add this and remove all the `box-sizing: border-box` we have across the codebase?